.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
```

Comment:
these don't seem to work in this repo (after merge)

Reply:
indeed, thanks for checking. Addressed in: #2

Reply:
+1, works now: https://github.com/jaegertracing/jaeger-ui/issues/new/choose